joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -608,29 +608,33 @@ CallbackReturn JointTrajectoryController::on_configure(const rclcpp_lifecycle::S
       return CallbackReturn::FAILURE;
     }
   }
-  else if (has_acceleration_state_interface_)
-  {
-    RCLCPP_ERROR(
-      logger,
-      "'acceleration' state interface cannot be used if 'position' and 'velocity' "
-      "interfaces are not present.");
-    return CallbackReturn::FAILURE;
-  }
-  else if (has_velocity_command_interface_ && command_interface_types_.size() == 1)
-  {
-    RCLCPP_ERROR(
-      logger,
-      "'velocity' command interface can only be used alone if 'velocity' and "
-      "'position' state interfaces are present");
-    return CallbackReturn::FAILURE;
-  }
-  else if (has_effort_command_interface_)
+  else
   {
-    RCLCPP_ERROR(
-      logger,
-      "'effort' command interface can only be used alone if 'velocity' and "
-      "'position' state interfaces are present");
-    return CallbackReturn::FAILURE;
+    if (has_acceleration_state_interface_)
```

Comment:
I think that changing this line should result in issues… Because there is no actual check for "position" and "velocity" interfaces. In the previous code, this was checked implicitly. So here should stay `else if`.